src/bridge/SequencerInbox.sol

```diff
@@ -469,11 +481,18 @@ contract SequencerInbox is GasRefundEnabled, ISequencerInbox {
     }
 
     /// @inheritdoc ISequencerInbox
-    function setIsSequencer(address addr, bool isSequencer_) external onlyRollupOwner {
+    function setIsSequencer(address addr, bool isSequencer_) external {
+        if (msg.sender != batchPosterManager) revert NotBatchPosterManager(msg.sender);
         isSequencer[addr] = isSequencer_;
         emit OwnerFunctionCalled(4);
```

Comment:
this (or if it changes, setIsBatchPoster) isn't an owner function anymore; should we change the event? (or at least add a comment?)

Reply:
hmm semantic, I prefer to keep the same event
```suggestion
        emit OwnerFunctionCalled(4); // Owner in this context can also be batch poster manager
```